desktop/core/src/desktop/lib/exceptions_renderable.py

```diff
@@ -83,3 +83,7 @@ def response(self, request):
       response.status_code = self.error_code
 
     return response
+
+class SlackBotException(PopupException):
```

Comment:
Nice!

As bot specific feel free to put it in the botserver views.py!

Reply:
Added in botserver views.py